shade_janitor/resources.py

```diff
@@ -54,43 +48,36 @@ def _add_floatingip(self, fip):
                         'floating_ip': fip.floating_ip_address,
                         })
 
-    def check_instance_blacklisted(self, instance):
-        skip_it = False
-        for entry in ['jenkins', 'slave', 'mirror']:
-            if entry in instance.name:
-                skip_it = True
-                break
-        return skip_it
+    def is_blacklisted(self, instance):
+        return instance.name in self.BLACKLIST
```

Comment:
This logic change breaks the searching logic as instance.name in self.BLACKLIST doesn't check substrings

old logic would check for blacklisted instances

jenkins-foo
mirror-local-bar

Which is intended to be skipped